libretro-common/string/stdstring.c

```diff
@@ -257,13 +252,114 @@ void word_wrap(char *dst, const char *src, int line_width, bool unicode, unsigne
 
             /* Early return if remaining src string
              * length is less than line width */
-            src_len    = (int)(src_end - src);
+            if (src_end - src < line_width)
+            {
+               strcpy(dst, src);
+               return;
+            }
+         }
+      }
+   }
+
+   *dst = '\0';
+}
+
+void word_wrap_wideglyph(char *dst, size_t dst_size, const char *src, int line_width, int wideglyph_width, unsigned max_lines)
+{
+   char *lastspace      = NULL;
+   char *lastwideglyph  = NULL;
+   unsigned counter     = 0;
+   unsigned lines       = 1;
+   int glyph_width      = line_width * 100;
```

Comment:
I would recommend renaming `glyph_width`. It is not actually the 'width of a glyph', and as such may confuse future developers who look at this code